### PR TITLE
Vet visits screen has been implemented

### PR DESCRIPTION
**Pull request by Afshin-Karamifar-Capgemini**

### CODE REVIEW

File: app/routes/vet-visits.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    auth: false,
+    handler: async (request, h) => {
+      const { organisation } = getFarmerApplyData(request)
+      const application = (
+        await getLatestApplicationsBySbi(organisation.sbi)
+      ).find((application) => {
+        return application.type === 'EE'
+      })
+
+      return h.view('vet-visits', {
```
Comment (alex742) [inline app/routes/vet-visits.js:22]: use the variable vetvisits from config

File: app/config/routes.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  vetVisits: 'claim/vet-visits'
```
Comment (alex742) [inline app/config/routes.js:2]: should be 'vet-visits' not 'claim/vet-visits'

File: test/integration/narrow/routes/vet-visits.test.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+jest.mock('../../../../app/session')
+jest.mock('../../../../app/api-requests/application-api')
+
+describe('Claim vet-visits', () => {
+  test('GET /claim/vet-visits route returns 200', async () => {
+    const options = {
+      method: 'GET',
+      url: '/claim/vet-visits',
+      auth: false
+    }
```
Comment (alex742) [inline test/integration/narrow/routes/vet-visits.test.js:15]: Will fail once auth is enabled.
A test for failure when no auth provided
A test for 200 when auth is provided - also check for content on page

File: app/routes/vet-visits.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  getLatestApplicationsBySbi
+} = require('../api-requests/application-api')
+
+const pageUrl = `/${vetVisits}`
+
+module.exports = {
+  method: 'GET',
+  path: pageUrl,
+  options: {
+    auth: false,
```
[the remark this answers is not in the document]
Comment (alex742) [reply app/routes/vet-visits.js:13]: The way to enable auth is to remove this line

File: app/routes/vet-visits.js (hunk 1 above)
Comment (Afshin-Karamifar-Capgemini) [reply app/routes/vet-visits.js:22]: Thanks for your comment. Done!

File: app/config/routes.js (hunk 2 above)
Comment (Afshin-Karamifar-Capgemini) [reply app/config/routes.js:2]: Thanks for your comment. Done!

File: app/routes/vet-visits.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (Afshin-Karamifar-Capgemini) [reply app/routes/vet-visits.js:13]: Thanks for your comment. Done!

File: test/integration/narrow/routes/vet-visits.test.js (hunk 3 above)
Comment (Afshin-Karamifar-Capgemini) [reply test/integration/narrow/routes/vet-visits.test.js:15]: Thanks for your comment. Done!

**Review by alex742 — APPROVED** [review]

**Pull request merged**